docs/UserGuide.md

```diff
@@ -155,38 +155,85 @@ This command removes the module stated in the command (“CS2103T”) from the l
 
 ### Tagging a module to contact : `tag`
 
-Adds a module tag to an existing contact.
-Format: `tag n/NAME m/MODULE_TAG`
-Example of usage: `tag n/John Doe m/CS2103T`
+Adds module tag(s) to an existing contact.
+
+Formats: 
+* `tag CONTACT_INDEX m/MODULE_TAG`
+* `tag m/MODULE_TAG`
```

Comment:
Basically added two formats for the command.
I expect to add two more in the future to showcase multiple commands.

Reply:
Ok.